.gitignore

```diff
@@ -0,0 +1,19 @@
+objs/
```

Comment:
why do some earn a /, and others do not?

Reply:
Quick and dirty conversion from the hgignore. No particular reason :)